docs/reference/tools.md

```
@@ -204,6 +204,146 @@ Full embedding vector returned with 768 dimensions.
 ```
 
 
+## list_database_connections
+
+The `list_database_connections` tool lists available database connections
+that the LLM can switch between. This tool is disabled by default and
+must be enabled in the server configuration.
+
+**Use Cases**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)